README.md

```diff
@@ -173,12 +173,14 @@ To view your coverage report:
 
 <img width="500" src="screen.png">
 
-you can use [any reporters that are supported by `istanbul`](https://github.com/istanbuljs/istanbuljs/tree/master/packages/istanbul-reports/lib):
+You can use [any reporters that are supported by `istanbul`](https://github.com/istanbuljs/istanbuljs/tree/master/packages/istanbul-reports/lib): `clover`, `cobertura`, `html`, `json-summary`, `json`, `lcov`, `lcovonly`, `none`, `teamcity`, `text-lcov`, `text-summary`, `text`.
 
 ```bash
 nyc report --reporter=lcov
 ```
 
+You can find examples of the output for various reporters [here](https://istanbul.js.org/docs/advanced/instrument/alternative-reporters).
```

Comment:
link should be: https://istanbul.js.org/docs/advanced/alternative-reporters/

once this is fixed we can land things, thanks for your updates to the docs 👍

Reply:
Oh silly me! I'll fix it straight away.